src/components/MainTable/MainTable.js

```diff
@@ -219,6 +229,7 @@ const MainTable = ({
 MainTable.propTypes = {
   defaultSort: PropTypes.string,
   defaultSortDirection: PropTypes.oneOf(["ascending", "descending"]),
+  emptyStateMsg: PropTypes.string,
```

Comment:
I think this should be `PropTypes.node` so that more complex content can be passed in as well. Maybe this should just be called `emptyState`. Also, might be good to add a doc string as well (like this: https://github.com/canonical-web-and-design/react-components/blob/master/src/components/Accordion/Accordion.js#L34), something like:

```js
/**
 * A state that will be shown when no rows are passed to the table.
 */
emptyState: PropTypes.node,
```